### ISSUE TITLE
Allow to cancel requests

### ISSUE THREAD

**Issue by tahini** [theia-ide/theia-trace-extension#55]
Add mechanism to cancel a request. Using the abort signal from the tsp-typescript-client and the listener in the Trace Compass server

**Comment by ebugden** [comment]
@bhufmann was this fixed via #259?

**Comment by bhufmann** [comment]
> @bhufmann was this fixed via #259?

No, #259 supports cancellation of the open procedure, i.e. scanning selected directory for CTF traces, posting trace and experiment to server and open the timeline container.

This issue tracker (#55) is to support cancellation of individual TSP REST calls and related handling of cancellation.
